### PR TITLE
[docs] Describe the concurrency feature

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @AndreyBelym

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -269,7 +273,26 @@ For example, the following command runs tests from the specified file and forces
...
+The following example shows how to run tests in three Chrome and three Firefox instances.
+
+```sh
+testcafe -c 3 chrome,firefox tests/sample-fixture.js
+```
+
+When you run tests on [remote devices](#remote-browsers),
+specify the total number of instances of all browsers you test against after the `remote:` keyword.
+On a remote device, invoke all the required instances manually. The total number of instances
+should divide by `factor`. Otherwise, an exception will be thrown.
```
Comment (helen-dikareva) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Let's add cmd example for using `remote` and `concurrency`

File: docs/articles/documentation/using-testcafe/programming-interface/runner.md [hunk 2]
```diff
@@ -266,6 +267,37 @@ runner.reporter('json', stream);
 
 You can also build your own reporter. Use a [dedicated Yeoman generator](https://github.com/DevExpress/generator-testcafe-reporter) to scaffold out a [reporter plugin](../../extending-testcafe/reporter-plugin/README.md).
 
+### concurrency
+
+Specifies that tests should run concurrently.
+
+```text
+async concurrency(factor) → this
```
Comment (helen-dikareva) [inline docs/articles/documentation/using-testcafe/programming-interface/runner.md]: it's should be  `runner.concurrency...`

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 3]
```diff
@@ -269,7 +273,26 @@ For example, the following command runs tests from the specified file and forces
 testcafe ie tests/sample-fixture.js -e
 ```
 
-### -t \<name\>,  --test \<name\>
+### -c \<factor\>, --concurrency \<factor\>
+
+Runs tests concurrently. The `factor` parameter specifies the number of browser instances
```
Comment (churkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: `Runs tests concurrently` what does it mean? Run all tests three times? Why?

Why do I need this? We must first say that this saves time of tests passing.

Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: I agree with @churkin.

The first two sentences are:
```js
Runs tests concurrently. ... the number of browser instances that will be invoked at the same time to run tests concurrently.
```
It's actual not for `concurrently mode` only. When we run just `testcafe chrome,firefox test.js` we run tests **concurrently** in two browsers. So, we should chose a wording that shows the difference clearly.

I think we can add a new section somewhere in the documentation that describes this mode in details. And we can post a link to it from the CLI API and Programming API sections.

**Comment by VasilyStrelyaev** [comment]
FPR

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 4]
```diff
@@ -269,7 +273,20 @@ For example, the following command runs tests from the specified file and forces
...
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: The problem is still actual. Discussed in private

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 5]
```diff
@@ -269,7 +273,20 @@ For example, the following command runs tests from the specified file and forces
...
+
+Specifies that tests should run concurrently. TestCafe will open `n` browser instances,
+divide tests in `n` groups and run each group in its own browser instance.
+
+To learn more about concurrent test execution, see [Concurrent Test Execution](common-concepts/concurrent-test-execution.md).
+
+The following example shows how to run tests in three Chrome and three Firefox instances.
+
+```sh
+testcafe -c 3 chrome,firefox tests/sample-fixture.js
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Please change it with the one-browser example (chrome) as we discussed before

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 6]
```diff
@@ -269,7 +273,20 @@ For example, the following command runs tests from the specified file and forces
 testcafe ie tests/sample-fixture.js -e
 ```
 
-### -t \<name\>,  --test \<name\>
+### -c \<n\>, --concurrency \<n\>
+
+Specifies that tests should run concurrently. TestCafe will open `n` browser instances,
+divide tests in `n` groups and run each group in its own browser instance.
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: > divide tests in `n` groups and run each group in its own browser instance

I have doubts about this sentence. Let's discuss it in private

**Pull request merged**